src/Common/Admin/Entities/Button.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Button element.
+ *
+ * @since 6.1.0
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Common\Admin\Entities;
+
+use Tribe\Utils\Element_Attributes as Attributes;
+use Tribe\Utils\Element_Classes as Classes;
+
+/**
+ * Class Button
+ *
+ * @since TBD
+ */
+class Button extends Container {
+
+	/**
+	 * Div constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param ?Classes    $classes    The classes for the div.
+	 * @param ?Attributes $attributes The attributes for the div.
+	 */
+	public function __construct( ?Classes $classes = null, ?Attributes $attributes = null ) {
+		if ( $classes ) {
```

Comment:
ugh, the parent functions should handle passing empty values - how do I _remove_ a class/attribute?

Reply:
@Camwyn 

> the parent functions should handle passing empty values

The reason parent doesn't have its own constructor was to allow for the different needs of child elements. But you could easily update them to handle only classes and attributes.

> how do I remove a class/attribute?

What is a scenario where you need to remove a class or attribute?